airflow/serialization/serialized_objects.py

```diff
@@ -53,14 +55,23 @@
 log = logging.getLogger(__name__)
 FAILED = 'serialization_failed'
 
-BUILTIN_OPERATOR_EXTRA_LINKS: List[str] = [
-    "airflow.providers.google.cloud.operators.bigquery.BigQueryConsoleLink",
-    "airflow.providers.google.cloud.operators.bigquery.BigQueryConsoleIndexableLink",
-    "airflow.providers.google.cloud.operators.mlengine.AIPlatformConsoleLink",
-    "airflow.providers.qubole.operators.qubole.QDSLink",
+_OPERATOR_EXTRA_LINKS: Set[str] = {
     "airflow.operators.dagrun_operator.TriggerDagRunLink",
     "airflow.sensors.external_task_sensor.ExternalTaskSensorLink",
-]
+}
+
+cache = lru_cache(maxsize=None)
+
+
+@cache
```

Comment:
```suggestion

@lru_cache(maxsize=None)
```

Reply:
This was because in newer versions of func_tools there is a `cache` function that does this, so it's mirroring that. Could probably do with a comment, or at the top:

```python
try:
    from functools import cache
except ImportError:
    from functools import lru_cache
    cache = lru_cache(maxsize=None)
```

Reply:
oh but the file just contains 1 import currently so didn't know:

```
from functools import lru_cache
```

Reply:
Will add some comments if I decide to leave the cache . Not sure yet :)

Reply:
Can you add a comment since we're keeping the cache?

Reply:
Added the conditional import. Should be good to go.